libs/components/router/testing/src/href/href-resolver-mock.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Inject, Injectable, InjectionToken } from '@angular/core';
+import { SkyHref, SkyHrefResolver } from '@skyux/router';
+
+export const MockUserHasAccess = new InjectionToken<boolean>(
+  'MockUserHasAccess'
+);
+
+@Injectable()
```

Comment:
I'd put `@internal` above this type so it's clear we don't intend this to be exported.

Reply:
Added `@internal`.